bedrock/styleguide/urls.py

```diff
@@ -51,9 +51,12 @@
     PageNode('Websites', path='websites', children=(
         PageNode('Sandstone', path='sandstone', children=(
             PageNode('Overview', template='styleguide/websites/sandstone-intro.html'),
+            PageNode('Buttons', path='buttons', template='styleguide/websites/sandstone-buttons.html'),
             PageNode('Colors', path='colors', template='styleguide/websites/sandstone-colors.html'),
+            PageNode('Forms', path='forms', template='styleguide/websites/sandstone-forms.html'),
             PageNode('Grids', path='grids', template='styleguide/websites/sandstone-grids.html'),
             PageNode('Tables & Lists', path='tables', template='styleguide/websites/sandstone-tables.html'),
+            PageNode('Tabzilla', path='tabzilla', template='styleguide/websites/sandstone-tabzilla.html'),
```

Comment:
Looks like these URLs (and templates) were removed in #4104. Maybe an accidental regression here?